app/views/subscriptions_management/confirm_unsubscribe_all.html.erb

```diff
@@ -1,4 +1,4 @@
-<% content_for :title, 'Are you sure you want to unsubscribe from everything?' %>
+<% content_for :title, t("subscriptions_management.confirm_unsubscribe_all.heading") %>
```

Comment:
Minor: I'm curious about the use of `heading`. In the past, I think we've used `title` everywhere. Doesn't really matter - just wondering if there's a reason for the choice of word?

Reply:
I think it's just the first place I extracted a translation was in a heading, and then spotted the same text was being used in the title, so just reused the translation I had already created.